rust/protocol/src/sealed_sender.rs

```diff
@@ -565,11 +565,13 @@ impl UnidentifiedSenderMessage {
             SEALED_SENDER_V2_VERSION => {
                 // Uses a flat representation: C || AT || E.pub || ciphertext
                 let remaining = &data[1..];
-                if remaining.len() < 32 + 16 + 32 {
+                if remaining.len() < sealed_sender_v2::M_LEN + sealed_sender_v2::AT_LEN + 32 {
                     return Err(SignalProtocolError::InvalidProtobufEncoding);
                 }
-                let (encrypted_message_key, remaining) = remaining.split_at(32);
-                let (encrypted_authentication_tag, remaining) = remaining.split_at(16);
+                let (encrypted_message_key, remaining) =
+                    remaining.split_at(sealed_sender_v2::M_LEN);
```

Comment:
Nit: MESSAGE_LEN would be a bit more readable

Reply:
It's not the message length, it's the seed for the keys used to encrypt the message. (I asked Trevor for more verbose names and he replied that there wasn't a standard term, and that cryptographers tend to use initials anyway.) Spelling out long constant names also makes them stand out a *lot* in this code (AUTHENTICATION_TAG_LEN).